src/Modal.js

```diff
@@ -204,6 +204,7 @@ class Modal extends React.Component {
 
     const focusableChildren = this.getFocusableChildren();
     const totalFocusable = focusableChildren.length;
+    if (totalFocusable == 0) return;
```

Comment:
strict equality check please (change `==` to `===`)

Reply:
done